src/Mod/OpenSCAD/exportCSG.py

```diff
@@ -269,4 +269,4 @@ def export(exportList,filename):
     csg.write("}\n}\n")
     # close file              
     csg.close()
-    FreeCAD.Console.PrintMessage("successfully exported "+filename)
+    FreeCAD.Console.PrintMessage("successfully exported" + " " + filename)
```

Comment:
first part should go into a translate()

Reply:
@yorikvanhavre there are no other translate()'s in the PrintMessage() in the rest of this file so maybe that would be a separate PR?

Reply:
sure!